### PR TITLE
PR for Readme.md

### PR DESCRIPTION
**Pull request by divyank96**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,940 @@
...
+                "BankName": "<<BANK_NAME>>",
+                "BankAccountNo": "<<ACCOUNT_NUMBER>>"
+              }
+            }
+          }
+        }
+      },
+      {
+        "2017-18": {
+          "ITR1FORM:ITR1": {
```
Comment (thefoodiecoder) [inline Readme.md]: I think the response format has changes @iAziz786 Can you share the correct format. `ITR1FORM:` is removed

**Review by thefoodiecoder — CHANGES_REQUESTED** [review]
Check response formats. I think they have changed.

File: Readme.md (hunk 1 above)
Comment (iAziz786) [reply Readme.md]: @thefoodiecoder Yes, I think it's an old response. @divyank96 can you please copy the latest from zoop-web-sdk.

Comment (divyank96) [reply Readme.md]: already did and raised the PR again can you please verify once @iAziz786 in develop branch

**Review by thefoodiecoder — APPROVED** [review]

**Pull request merged**